doc/Yang Model Guidelines.md

```diff
@@ -1,20 +1,23 @@
-#   YANG MODEL GUIDELINES
+#   SONiC YANG MODEL GUIDELINES
 
 ### Rev 1.0 ###
 
 ### Revision ###
 
  | Rev |     Date    |       Author       | Change Description                |
  |:---:|:-----------:|:------------------:|-----------------------------------|
- | 1.0 | 22 Aug 2019             |      Praveen Chaudhary      | Initial version                   |
+ | 1.0 | 22 Aug 2019             |      Praveen Chaudhary      | Initial version                    |
+ | 1.1 | 29 Aug 2019             |      Praveen Chaudhary      | Addressed review comments.
 
 
 
-## This document lists the guidelines, which will be used to write Yang Modules for Sonic. These Yang Modules will be primarily based on ABNF Schema of Sonic. 
+###  This document lists the guidelines, which will be used to write YANG Modules for SONiC. These YANG Modules will be primarily based on or represent the ABNF Schema of SONiC, and the syntax of YANG models must follow RFC 7950.
```

Comment:
We should also  clearly state where user should start from - ABNF schema and/or ABNF JSON i.e. Redis JSON data in config_db.json.